libtado/api.py

```diff
@@ -54,20 +56,27 @@ def __init__(self, username, password, secret):
 
   def _login(self):
     """Login and setup the HTTP session."""
+    print(pkg_resources.require)
```

Comment:
I would advise removing the print statements after confirming the user-agent is correctly sent in the requests.

Reply:
Refactoring is on the way